lookup/lookup.js

```diff
@@ -10,8 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-var request = require('request');
-var async = require('async');
```

Comment:
was async an npm dep? If so, do we have to delete it from a package.json?

Reply:
Hmm, it wasn't in `package.json` before. Perhaps it was a package provided by Adobe Runtime? I can't find docs on it (fkn adobe.. argh!) - @hirenoble do you know?